content/_data/upgrades/2-346-1.adoc

```diff
@@ -0,0 +1,30 @@
+==== PNG and GIF icons replaced with SVG icons
+
+Scalable vector graphics images are used in Jenkins instead of the GIF and PNG images that were used previously.
+SVG images provide a better user experience and improve the Jenkins user experience.
+
+The GIF and PNG images have been removed from Jenkins core.
+
+Jenkins has removed a large number of GIF and PNG iamges.
+Ensure all your plugins are the most recent release before updating core, otherwise images may be missing.
+Check for new plugin versions immediately after the core update and install available updates.
+
+==== Java Native Runtime library removed
+
+The Java Native Runtime (JNR) library has been removed from Jenkins core.
+Users of https://plugins.jenkins.io/pam-auth/[PAM Authentication], https://plugins.jenkins.io/file-leak-detector/[CloudBees File Leak Detector], and https://plugins.jenkins.io/maven-repo-cleaner/[Maven Repository Scheduled Cleanup] must upgrade those plugins to compatible versions.
```

Comment:
Ideally this would list the releases to update to.

Reply:
Updated in e17fa852fc3911ada87061260219899674f9d69e